docs/_data/portland-2025-schedule.yaml

```
@@ -0,0 +1,156 @@
+outing:
+  - time: '13:45'
+    title: Arrive at trailhead
+  - time: '14:00'
+    title: Hike starts
+  - time: '17:00'
+    title: Hike ends - end time approximate
+
+writing_day:
+  - time: '8:00'
+    title: 'Doors open'
+  - time: '08:45'
+    title: 'Project leaders select and set up at available tables'
+  - time: '9:00'
+    title: Morning introduction - Introduction to Sprints & Project Introductions
+  - time: '9:30'
+    title: Team up and start working
+  - time: '11:50'
+    title: '<i>Welcome Wagon conference introduction and venue tour</i>'
+  - time: '12:00'
+    title: Lunch Break
+  - time: '13:30'
+    title: '<i>Welcome Wagon conference introduction and venue tour</i>'
+  - time: '13:45'
+    title: 'Afternoon Session - all day groups reconvene and new group introductions'
+  - time: '16:30'
+    title: '<i>Welcome Wagon conference introduction and venue tour</i>'
+  - time: '17:00'
+    title: '<b>End of Writing Day</b>'
+  - time: '17:00'
+    title: '<b>Reception starts</b>'
+  - time: '19:00'
+    title: '<b>Reception ends</b>'
+
+talks_day1:
+  - time: '8:00'
+    title: Doors open
+  - time: '8:15'
+    title: '<i>Welcome Wagon conference introduction and venue tour</i>'
+  - time: '8:30'
+    title: '<b>Sponsor booths open</b>'
+  - time: '9:00'
+    duration: '0:25'
+    title: Write the Docs team - Conference introduction
+  - duration: '0:05'
+    title: '5 minute break'
+  - duration: '0:00'
+    title: "<b>Unconference starts</b>"
+  - duration: '0:30'
+    slug: quest-for-the-holy-grail-turning-user-feedback-into-meaningful-change-rachel-rigdon
+  - duration: '0:10'
+    title: '✋ 10 min Q&A'
+  - duration: '0:05'
+    title: '5 minute break'
+  - duration: '0:30'
+    slug: the-power-of-two-how-support-and-documentation-teams-transform-technical-conten-yanjie-niu
+  - duration: '0:5'
+    title: 'Sponsor intros'
+  - duration: '0:10'
+    title: '✋ 10 min Q&A'
```

Feels weird to have this before the Q&A? 

```suggestion
  - duration: '0:10'
    title: '✋ 10 min Q&A'
  - duration: '0:5'
    title: 'Sponsor intros'
```

---

Yep, that was straight up human error.